packages/razorsharp/README.md

```diff
@@ -0,0 +1,49 @@
+# Razorsharp
+
+> **Warning**
+>
+> This is alpha software. Things may break and code may not be generated correctly. Please report any issues you find.
+
+Razorsharp is a Figma plugin that generates code for the Blade design system.
+
+## Watch a demo
+
+[![Watch the video](https://img.youtube.com/vi/1_Va4O9kDp8/maxresdefault.jpg)](https://youtu.be/1_Va4O9kDp8)
+
+## Screenshots
+
+<p float="left">
+    <img src="./docs/images/screenshot-1.png" width="200px" height="auto">
+    <img src="./docs/images/screenshot-2.png" width="200px" height="auto">
+    <img src="./docs/images/screenshot-3.png" width="200px" height="auto">
+</p>
+
+## What this is not
+
+Do not expect this plugin to generate 100% production ready code. To get the most out of this plugin:
+
+- Ask designers to always use Blade components in Figma
+- Ask designers to use Auto Layout in Figma
+- Generate code per design section instead of the whole page
+
+## Usage guide
```

Comment:
Would be nice to have some sort of high level overview of "How it works" documentation. Otherwise hard to picture how it's doing things internally just by looking at the code.

Reply:
Agreed. Both "How it works" and "Contributing" are currently missing